Tests/js/models/assets/ez-usermodel-tests.js

```diff
@@ -28,9 +30,21 @@ YUI.add('ez-usermodel-tests', function (Y) {
                     "login": "admin",
                     "email": "dp@ez.no",
                     "enabled": "true",
-                    "name": "Administrator User"
-                }
+                    "name": "Administrator User",
+                    "Version":  {
+                        "Fields": {
+                            "field": []
```

Comment:
since this array is always empty, the filter function in `_parseStruct` is not covered and the alternative right after it is half covered as well.